GettingStarted.md

```diff
@@ -128,6 +116,19 @@ $ ip addr
 <snip />
 ```
 
+## Container Development Kit (CDK) Walkthrough
+
+The cool thing is, we actually launched the CDK VM by launching the above Vagrantfile. The CDK gives a couple options (through the use of [different Vagrantfiles](https://developers.redhat.com/download-manager/file/cdk-2.0.0.zip)) but in this case, we launched the "OpenShift in a Box" VM (aka rhel-ose in the cdk.zip). The CDK provides a simple to use and launch instance of the same OpenShift PaaS you would use at work. Why is a PaaS included in a product, much less a lab, focused on Containers? Well, the latest version of OpenShift, actually runs Docker Containers to host your "Platform" (in the PaaS sense) and your application.
+
+If you would like to explore the OpenShift Console, you can see it running in your OpenShift instance, if you open a browser. Let's go ahead and try it. Open Firefox from the Applications menu and navigate to `https://10.1.2.2:8443/console/`. Once it loads (and you bypass the bad certificate error), you can log in to the console using the default `admin/admin` or to see the less privileged experience, use `openshift-dev/devel` for the `username/password`.
+
+#### the following example doesn't seem to work for me in the VM (xip.io??)
```

Comment:
This example won't work offline because the sample application pulls from a github repository I believe.